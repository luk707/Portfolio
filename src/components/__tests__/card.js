// @flow

import React from 'react'
import renderer from 'react-test-renderer'
import { ThemeProvider } from 'emotion-theming'
import Card from '../card'
import defaultTheme from '../../theme'

describe('card', () => {
  it('renders as expected', () => {
    const tree = renderer
      .create(
        <ThemeProvider theme={defaultTheme}>
          <Card>This is a test card</Card>
        </ThemeProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
