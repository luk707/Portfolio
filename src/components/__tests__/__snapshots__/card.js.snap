// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card renders as expected 1`] = `
<div
  className="css-amd3ig"
>
  This is a test card
</div>
`;
